### ISSUE TITLE
Using local endpoints for testing

### ISSUE THREAD

**Issue by j0ni** [cognitect-labs/aws-api#43]
A question, and possibly a feature request:

tl;dr Is there a "correct" way to substitute a local endpoint for an AWS hosted endpoint?

I work on applications which make use of several AWS products - for our CI tests, we use docker-compose with services which are (for our purposes) functionally identical behaviours. The SDKs we currently use all allow us to configure endpoint URLs directly. I cannot see an obvious hook for this here. I looked at first at simply shadowing the `endpoints.edn` file, but it isn't obvious to me where that comes from, or lives, or is supposed to look like. So before pursuing that line, I figured I find out if there's a simpler way.

If this is not currently a supported use case, I would like to request it be added.

Thanks for your work on this @dchelimsky

**Comment by ghadishayban** [comment]
Some services like CloudSearch require an explicit endpoint, too.
